extensions/mailer/runtime/src/main/java/io/quarkus/mailer/runtime/MailConfig.java

```diff
@@ -74,8 +75,8 @@
      * Configures the maximum allowed number of open connections to the mail server
      * If not set the default is {@code 10}.
```

Comment:
You could remove this now that you have a default value. It's automatically documented.